test/e2e/recurly.test.js

```diff
@@ -100,6 +100,52 @@ describe('Recurly.js', async () => {
     });
   });
 
+  describe('credit card with tax identifier', async function () {
+    beforeEach(init({ fixture: 'hosted-fields-card-tax-identifier' }));
+
+    const sel = {
+      output: '[data-test=output]',
+      form: '[data-test=form]',
+      submit: '[data-test=submit]',
+      firstName: '[data-test="first-name"]',
+      lastName: '[data-test="last-name"]',
+      taxIdentifier: '[data-test="tax-identifier"]',
+      iframe: '.recurly-hosted-field iframe',
+      number: 'input[placeholder="Card number"]',
+      expiry: 'input[placeholder="MM / YY"]',
+      cvv: 'input[placeholder="CVV"]'
+    };
```

Comment:
Feel free to move the existing `const sel` scope up to a level where you can access it.